Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/batteryInput/index.tsx

```diff
@@ -1,4 +1,15 @@
-const BatteryInput = () => {
+import React, {FC, InputHTMLAttributes} from "react";
+
+interface BatteryInputProps {
+  value: number; // Specify the type as 'number'
+  onChange: (newValue: number) => void; // Define the type for the 'onChange' callback
+}
+
+const BatteryInput: React.FC<BatteryInputProps> = ({ value, onChange }) => {
```

Comment:
Incorrect syntax here. Two issues:
1. React.FC doesn't support generics.
2. React.FC doesn't work correctly with defaultProps.

React.FC is preferred when we need to access to typed `children`.
In this teaser we aren't rendering children within the 3 input components, so just write it like this (actually the preferred way in 99% of scenarios)

const BatteryInput =  ({ value, onChange }: BatteryInputProps ) => {...}